marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.test.tsx

```diff
@@ -5,18 +5,35 @@ import ManageQuery from './manage-query';
 
 describe('Query Modal Component', () => {
 
-    test('Modal is not visible', () => {
-      const { queryByTestId } =  render(<ManageQuery />);
+  let query = [{ "savedQuery": { "id": "00529904-ed6e-4539-b7d8-eeddaf15aaa4", "name": "Lee", "description": "", "query": { "searchText": "", "entityTypeIds": ["Customer"], "selectedFacets": { "firstname": { "dataType": "xs:string", "stringValues": ["Lee"] } } }, "propertiesToDisplay": ["id"], "owner": "admin", "systemMetadata": { "createdBy": "admin", "createdDateTime": "2020-05-05T14:48:53.206721-07:00", "lastUpdatedBy": "admin", "lastUpdatedDateTime": "2020-05-05T14:48:53.206721-07:00" } } }]
 
-      expect(queryByTestId('manage-queries-modal')).toBeNull();
-    });
+  test('Modal is not visible', () => {
+    const { queryByTestId } = render(<ManageQuery />);
+    expect(queryByTestId('manage-queries-modal')).toBeNull();
```

Comment:
I will approve this now. Just a thought for future tests we add, that the 2 modal visibility tests can be combined with a description "Verify modal visibility" and the assertion for not visible until you click on the modal icon can be moved into that test.